### DISCUSSION TITLE
peso valutazione

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#650] [Idee e proposte]
Buongiorno,sarebbe possibile implementare la funzione del peso che può avere un voto , in modo da decidere in base alla prova se sitratta di un voto che va ad incidere sulla media del 40%,del 50% ,100% etc?Grazie mille, sono tutte idee , se si possono realizzare va bene ,altrimenti va bene lo stesso

**Comment by trinko** [comment]
Per il momento la segno come richiesta. Vediamo in seguito.
